lib/src/main/java/org/team100/lib/config/NotePoseDetector.java

```diff
@@ -0,0 +1,97 @@
+package org.team100.lib.config;
+
+import java.util.Optional;
+
+import org.team100.lib.localization.NotePosition24ArrayListener;
+import org.team100.lib.motion.drivetrain.SwerveDriveSubsystem;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Transform2d;
+import edu.wpi.first.math.geometry.Translation2d;
+
+public class NotePoseDetector {
```

Comment:
it's not clear to me that this class needs to exist at all.  it seems like it just does a tiny bit of geometry, which is only used by the command?  maybe extract the geometry as a static method in a utility class like PoseEstimatorHelper and get rid of this class